kuzzleio/kuzzle-admin-console
# fix users not displayed

## What does this PR do ?
Fix #767 users not displayed when url contains from=XXX and there are less than XXX users in kuzzle by forcing AC to set from=0

### How should this be manually tested?
  - Step 1 : Create an user
  - Step 2 : go to `http://localhost:8080/#/security/users?from=10`
  - Step 3 : check if from=10 disappear from url and user is displayed


### Other changes
/

### Boyscout
/
